code/modules/mob/living/bot/medbot.dm

```diff
@@ -222,12 +222,19 @@
 		else
 			to_chat(usr, "<span class='notice'>You cannot eject the beaker because the panel is locked.</span>")
 
-	else if (href_list["togglevoice"] && (!locked || issilicon(usr)))
-		vocal = !vocal
-
 	else if (href_list["declaretreatment"] && (!locked || issilicon(usr)))
 		declare_treatment = !declare_treatment
 
+	else if (href_list["msg"] && (!locked || issilicon(usr)))
+		var/I = input(usr,"What will this medbot say?", "Set Message") as text|null
+		if(!I)
```

Comment:
This needs to be fed through one of the sanitization procs. Otherwise this can be used as a vector for HTML/JS injection and that is very bad juju.